### PR TITLE
Add FastSync steps to Sync FAQ

### LINKED ISSUE #209: Add Fast Sync to Sync FAQ
https://docs.btcpayserver.org/faq-and-common-issues/faq-synchronization

### PR DESCRIPTION
**Pull request by britttttk**
closes #209

### CODE REVIEW

File: FAQ/FAQ-Synchronization.md [hunk 1]
```diff
@@ -17,7 +17,21 @@ It may seem tedious, but it's a critical step of running your own full node and
 
 ## Can I skip the synchronization?
 
-You can't skip synchronization, but you can drastically decrease sync time. If you're comfortable with using the command line, you can use [FastSync](https://github.com/btcpayserver/btcpayserver-docker/tree/master/contrib/FastSync) to synchronize your node faster. You can also follow [this video](https://youtu.be/VNMnd-dX9Q8?t=1730). However, be sure to read the Fast Sync document to understand the potential trust issues involved with Fast Sync. 
+You can't skip synchronization, but you can drastically decrease sync time. If you're comfortable with using the command line, you can use FastSync to synchronize your node faster. Be sure to [read this FastSync document](https://github.com/btcpayserver/btcpayserver-docker/tree/master/contrib/FastSync) to understand the potential trust issues involved with FastSync.
+
+To use FastSync, [ssh into](FAQ-ServerSettings.md#how-to-ssh-into-my-btcpay-running-on-vps) your BTCPayServer instance and run the following commands:
+```
+sudo su -
+cd /root/btcpayserver-docker/
```
Comment (pavlenex) [inline FAQ/FAQ-Synchronization.md]: Have you tried these commands?
For me:
`cd /root/btcpayserver/btcpayserver-docker` does not work. You don't even need that step

This works
```
sudo su -
./btcpay-down.sh
cd contrib
cd FastSync
./load-utxo-set.sh
then ./btcpay-up.sh
```

File: FAQ/FAQ-Synchronization.md [hunk 2]
```diff
@@ -17,7 +17,21 @@ It may seem tedious, but it's a critical step of running your own full node and
 
 ## Can I skip the synchronization?
 
-You can't skip synchronization, but you can drastically decrease sync time. If you're comfortable with using the command line, you can use [FastSync](https://github.com/btcpayserver/btcpayserver-docker/tree/master/contrib/FastSync) to synchronize your node faster. You can also follow [this video](https://youtu.be/VNMnd-dX9Q8?t=1730). However, be sure to read the Fast Sync document to understand the potential trust issues involved with Fast Sync. 
+You can't skip synchronization, but you can drastically decrease sync time. If you're comfortable with using the command line, you can use FastSync to synchronize your node faster. Be sure to [read this FastSync document](https://github.com/btcpayserver/btcpayserver-docker/tree/master/contrib/FastSync) to understand the potential trust issues involved with FastSync.
+
+To use FastSync, [ssh into](FAQ-ServerSettings.md#how-to-ssh-into-my-btcpay-running-on-vps) your BTCPayServer instance and run the following commands:
+```
```
Comment (NicolasDorier) [inline FAQ/FAQ-Synchronization.md]: ` ```bash ` so there is proper formatting

File: FAQ/FAQ-Synchronization.md (hunk 1 above)
Comment (NicolasDorier) [reply FAQ/FAQ-Synchronization.md]: Better should be `cd $BTCPAY_BASE_DIRECTORY/btcpayserver-docker/`

File: FAQ/FAQ-Synchronization.md [hunk 3]
```diff
@@ -17,7 +17,21 @@ It may seem tedious, but it's a critical step of running your own full node and
...
+sudo su -
+cd /root/btcpayserver-docker/
+btcpay-down.sh
+cd contrib/FastSync
+./load-utxo-set.sh
+```
+After Fast Sync has completed:
+```
```
Comment (NicolasDorier) [inline FAQ/FAQ-Synchronization.md]: ` ```bash `

**Comment by NicolasDorier** [comment]
@britttttk you should mention that the user MUST use one of the `opt-save-storage` else bitcoind will be stuck in a restart loop.

File: FAQ/FAQ-Deployment.md [hunk 4]
```diff
@@ -18,7 +18,7 @@ Here are common questions about installation, regardless of the deployment metho
 * [How do I activate Tor on my BTCPay Server?](FAQ-Deployment.md#how-do-i-activate-tor-on-my-btcpay-server)
 * [How do I disable Tor on my BTCPay Server?](FAQ-Deployment.md#how-do-i-disable-tor-on-my-btcpay-server)
 * [Why activate Tor? Does it mean that nobody knows who I am?](FAQ-Deployment.md#why-activate-tor-does-it-mean-that-nobody-knows-who-i-am)
-* [How can I modify/deactivate environment variables?](FAQ-Deployment.md#How-can-i-modifydeactivate-environment-variables)
```
Comment (NicolasDorier) [inline FAQ/FAQ-Deployment.md]: should be `how-can-i-modify-deactivate-environment-variables` instead, the anchor is not correct.

File: FAQ/FAQ-Synchronization.md [hunk 5]
```diff
@@ -19,7 +19,7 @@ It may seem tedious, but it's a critical step of running your own full node and
 
 You can't skip synchronization, but you can drastically decrease sync time. If you're comfortable with using the command line, you can use FastSync to synchronize your node faster. Be sure to [read this FastSync document](https://github.com/btcpayserver/btcpayserver-docker/tree/master/contrib/FastSync) to understand the potential trust issues involved with FastSync.
 
-To use FastSync, make sure your deployment has a pruning option enabled by using an `opt-save-storage` [environment variable](FAQ-Deployment.md#how-can-i-modify-deactivate-environment-variables), otherwise bitcoind will not be able to sync. First step is to [ssh into](FAQ-ServerSettings.md#how-to-ssh-into-my-btcpay-running-on-vps) your BTCPayServer instance and run the following commands:
```
Comment (NicolasDorier) [inline FAQ/FAQ-Synchronization.md]: wrong anchor should be `how-can-i-modify-deactivate-environment-variables`

**Pull request merged**